### PR TITLE
[WIP] "InstalledFor" doesn't work for taskcluster org. Solution v.01

### PR DESCRIPTION
**Pull request by owlishDeveloper**
This is to try and remedy "Installed for" API method. I am trying to glue all the pages together. I'm not sure if it's quite this simple... For a start I want to see if I use hasNextPage() and getNextPage() correctly.

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -354,10 +354,19 @@ api.declare({
...
       let instGithub = await this.github.getInstallationGithub(ownerInfo.installationId);
       let reposList = await instGithub.integrations.getInstallationRepositories({});
+      debug(`REPOSLIST: ${JSON.stringify(reposList)}`);
 
       // GitHub API returns an array of objects, each of wich has an array of repos
       let installed = reposList.repositories.map(repo => repo.name).indexOf(repo);
 
+      if (installed === -1 && instGithub.hasNextPage()) {
+        while (installed === -1 && instGithub.hasNextPage()) {
+          reposList = instGithub.getNextPage();
```
Comment (djmitche) [inline src/api.js]: I think you need an `await` here!

Comment (owlishDeveloper) [reply src/api.js]: right!

Comment (owlishDeveloper) [reply src/api.js]: updated that, thanks!

**Comment by owlishDeveloper** [comment]
I corrected the logic, now it should work properly. However, before merging, I'd deploy the thing to regular tc-gh (not staging), to check the pagination stuff. I leave all the debugging statements for now. (I'll be away the first half of Monday. I'll ping you, @imbstack, on IRC somewhere in the afternoon so we could test it?)

**Comment by imbstack** [comment]
This work will continue in #188.

**Pull request closed without merging**